controllers/masteruserrecord/masteruserrecord_controller_test.go

```diff
@@ -122,6 +122,8 @@ func TestCreateUserAccountSuccessful(t *testing.T) {
 	logf.SetLogger(zap.New(zap.UseDevMode(true)))
 	s := apiScheme(t)
 	mur := murtest.NewMasterUserRecord(t, "john")
+	mur.Spec.OriginalSub = "original-sub:12345"
```

Comment:
Shouldn't we also add an assertion for the UserAcount bellow to make sure it does has the OriginalSub set?

Reply:
I added this assertion in toolchain-common, see https://github.com/codeready-toolchain/toolchain-common/blob/master/pkg/test/useraccount/assertion.go#L69

Reply:
Ah right..